projects/packages/forms/src/blocks/contact-form/util/with-shared-field-attributes.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { useDispatch, useSelect } from '@wordpress/data';
+import { select, useDispatch } from '@wordpress/data';
```

Comment:
I think you can still stick with `useSelect`, but there's a short form that doesn't have the dependencies or the mapping function:

```js
const { getBlockParentsByBlockName, getClientIdsOfDescendants, getBlocksByClientId } =
		useSelect( 'core/block-editor' );
```

Reply:
I started there and ended up talking myself out of it. Appreciate the explanation 👍 

I'll revisit this once I sort out a couple of other fixes for undo stack etc.

Reply:
Done in https://github.com/Automattic/jetpack/pull/41708/commits/115020721fa7d0b121d96c2c57c1ef2b5f8a2efb